themes/qgrid/layout/partials/nav.ejs

```diff
@@ -1,89 +1,27 @@
 <nav>
-    <h3>Getting started</h3>
-
     <ul>
-        <li>
-            <a href="#">Introduction</a>
-        </li>
-        <li>
-            <a href="#">How to install</a>
-        </li>
-        <li>
-            <a href="#">Concepts and architecture</a>
-        </li>
-        <li>
-            <a href="#">Grid model and service</a>
-        </li>
-    </ul>
+        <% var groupOrder = ['Getting started', 'API', 'Features'];
+        var groupped = site.pages.reduce(function (groups, p) {
+            var key = p.group;
+            (groups[key] = groups[key] || []).push(p);
+            return groups;
```

Comment:
usually we always use `memo` name as an aggregation result

Reply:
I'll rename, but prefer more significant names